department-of-veterans-affairs/vets-website
# MHV-53477 Use-The-Local-Title-As-The-Note-Title

## Summary

Revised the care summary and notes list as well as the details page to utilize record.content[0].attachment.title whenever it's accessible. If the title isn't available, the default to record.type.coding[0].display.
 
## Related issue(s)

[MHV-53477](https://jira.devops.va.gov/browse/MHV-53477) - Use the "Local Title" as the Note title

User Story: As a Medical Records Team, we want to map the data to the correct fields. so that the correct data is showing when the user views that data.

GIVEN

WHEN

THEN

Feature Flag Y/N ? N

Manual Testing Y/N ? N 

Automated Testing Y/N ? Y-Timothy

Accessibility Testing Y/N ? N

UCD Validation Y/N N

Acceptance Criteria

AC1  Parse the Local title from DocumentReference.content.title as the record title

AC2 If no local title is present use current functionality

AC3
AC4
AC5

Links to UCD:

Sketch Link:

User Flow:

Mobile:

Desktop:

Other Design Notes:

Architectural Notes: Always check for the latest designs in Figma

## Testing done
Conducted UI testing to ensure the expected behavior. I updated existing unit test for new components.

## Screenshots
![Screenshot 2024-02-08 at 3 44 20 PM (2)](https://github.com/department-of-veterans-affairs/vets-website/assets/8038703/55157431-d4a0-4322-a663-dfbcb302c1ca)


## What areas of the site does it impact?

Care summaries and notes - list and Details page

## Acceptance criteria

AC1  Parse the Local title from DocumentReference.content.title as the record title

AC2 If no local title is present use current functionality

### Quality Assurance & Testing


### Authentication
